src/griffe/docstrings/numpy.py

```diff
@@ -648,6 +665,7 @@ def parse(
     *,
     ignore_init_summary: bool = False,
     trim_doctest_flags: bool = True,
+    allow_block_breaks: bool = False,
```

Comment:
Shouldn't it be true to be prevent a breaking change? By the way I'm confused by the parameter name and description. I keep going back to the logic above to understand what it does. I think naming it `allow_section_blank_line` would be better :thinking: If true, then one needs *two* blank lines to break a section. WDYT?

Reply:
Hey, sorry for the wait--I was out of town last week. Agreed the parameter name is confusing, lemme go ahead and switch it to `allow_section_blank_line` and fix the default.

Reply:
No problem! Lets not be sorry for life things ^^ Thanks for the follow-up :)